scalac-scoverage-runtime/jvm/src/test/scala/scoverage/InvokerConcurrencyTest.scala

```diff
@@ -33,7 +32,7 @@ class InvokerConcurrencyTest extends FunSuite with BeforeAndAfter {
       Future {
         Invoker.invoked(i, measurementDir.toString)
       }
-    }(breakOut)
+    }.toList
```

Comment:
Actually we don't need a list here, set would be good (`val futures: Set[Future[Unit]] ...`)